plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -457,6 +457,8 @@ SecRule REQUEST_FILENAME "@unconditionalMatch" \
     pass,\
     t:none,\
     nolog,\
+    ctl:ruleRemoveTargetById=920273;ARGS_NAMES:users[0],\
+    ctl:ruleRemoveTargetById=942432;ARGS_NAMES:users[0],\
```

Comment:
These should not go here. I suggest to create a new rule for `/wp-admin/edit-tags.php` with `action=editedtag`.

Reply:
Hm, parameter `users[0]` is not part of issue #57, why are you adding it?

Reply:
it's from one of the other tests, I had to disable it to avoid the tests failing since I updated the rule IDs to check for in the tests.

Also, it's a parameter name so I'm fine disabling it globally. The rules will only be disabled for the exact parameter name.

Reply:
I don't get it. It is there only for tests to work?

Reply:
Yes, it's because all of the tests are looking for 920273 and 942432 now.

Reply:
@azurit is everything's good here?

Reply:
Ping @azurit

Reply:
I still don't get it. What is the test ID?

Reply:
@azurit if you check the `no_log_contains` in the tests you'll see additional rules are being checked for. These false positives were always there but were being ignored silently since the tests weren't looking for these rules.

Reply:
So let's remove them from tests, why it's there? I really don't get it at all.

Reply:
@azurit because there false positives? if we already know about them then we might as well fix it.

Reply:
@EsadCetiner But you said that these two lines (above) are only there for tests to pass. I'm lost.

Reply:
@azurit Both can be true. I updated the `no_log_contains` to check for additional rule IDs, the tests then failed when I updated the tests because before it was failing silently and not picking up the false positive. I then updated rule `9507350` to fix the newly found false positives that were being silently ignored before.

Reply:
@azurit I'd have to create another rule just to whitelist `ARGS_NAMES:users[0]`, it's simpler to keep them here, it'll be just as secure and probably faster.

Reply:
@EsadCetiner For which endpoints is `ARGS_NAMES:users[0]` doing FPs?

Reply:
I've moved it to a new rule